packages/sdds-cs/src/components/Calendar/Calendar.stories.tsx

```diff
@@ -268,14 +268,14 @@ const StoryRange = (args: CalendarBaseRangeProps & { displayDouble: boolean }) =
     return (
         <>
             <H3>Тип календаря</H3>
-            <Tabs view="default" size="m">
+            <Tabs view="divider" size="m" hasDivider>
```

Comment:
Не ломает пример, но предлагаю поправить, чтобы точно в будущем не стрельнуло

```suggestion
            <Tabs view="divider" size="s" hasDivider>
```